sdk/monitor/Azure.ResourceManager.Monitor/tests/TestCase/MetricAlertCollectionTests.cs

```diff
@@ -14,10 +15,34 @@ namespace Azure.ResourceManager.Monitor.Tests
     public class MetricAlertCollectionTests : MonitorTestBase
     {
         public MetricAlertCollectionTests(bool isAsync)
-           : base(isAsync)
+           : base(isAsync)//, RecordedTestMode.Record)
         {
         }
 
+        #region storage account
+        public async Task<string> GetStorageAccountId()
+        {
+            var resourceGroup = await CreateResourceGroupAsync();
+            var storageCeollection = resourceGroup.GetStorageAccounts();
+            string accountName = Recording.GenerateAssetName("metrictests");
+            var storageContent = ResourceDataHelper.GetContent();
+            var storageAccount = await storageCeollection.CreateOrUpdateAsync(WaitUntil.Completed, accountName, storageContent);
+            string storageAccountId;
+            if (Mode == RecordedTestMode.Playback)
+            {
+                storageAccountId = $"/subscriptions/db1ab6f0-4769-4b27-930e-01e2ef9c123c/resourceGroups/{resourceGroup.Id.Name}/providers/Microsoft.Storage/storageAccounts/{accountName}";
```

Comment:
Please use the `StorageAccountResource.CreateResourceIdentifier` method, and use `resourceGroup.Id.SubscriptionId` for susbcriptionId instead of hard coding it here.

Reply:
fixed